core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -93,7 +93,7 @@ public void setUp(@TempDir Path tmpFolder) throws Exception {
     Path loginPropertiesFile = tmpFolder.resolve("login.properties");
 
     String jaas = "c3 {\n"
-                  + "  org.eclipse.jetty.jaas.spi.PropertyFileLoginModule required\n"
+                  + "  org.eclipse.jetty.security.jaas.spi.PropertyFileLoginModule required\n"
```

Comment:
Are our users going to have to change this jaas string too in CP?

Reply:
yeah I think so.. we should make a note for the release